packages/office-addin-debugging/src/defaults.ts

```diff
@@ -15,4 +15,14 @@ export const usageDataOptions: usageData.IUsageDataOptions = {
     usageDataLevel: sendUsageData ? usageData.UsageDataLevel.on : usageData.UsageDataLevel.off,
     method: usageData.UsageDataReportingMethod.applicationInsights,
     isForTesting: false
-}
+};
+const usageDataObject: usageData.OfficeAddinUsageData = new usageData.OfficeAddinUsageData(usageDataOptions);
+export function sendUsageDataSuccessEvent(method: string, ...data: object[]) {
+    usageDataObject.sendUsageDataSuccessEvent(usageDataProjectName, data.concat({Method: method}));
+};
+export function sendUsageDataException(method: string, error: Error | string, ...data: Object[]) {
```

Comment:
I think using a properties object is preferrable to a variable number of arguments. The caller would create the object with properties rather than just pass additional args. This is a better javascript way to do it here. Glad to explain more in person.

You should also use "object" and not Object for types.